CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -166,15 +171,24 @@ class EditingViewController: UIViewController {
 
     private func setupSections() {
         if isShowingGroups {
-            setupSectionsFomGroups()
+//            setupSectionsFomGroups()
+            let organizationStrings = Constants.Options.organizations.map { $0.name }
+            let yoursAndMore = removeDuplicates(yourStrings: user.groups, moreStrings: organizationStrings)
+            let stringToGroup = { ItemType.group(Constants.Options.organizationsMap[$0]!) }
+            sections = [
+               Section(type: .yours, items: yoursAndMore.your.map(stringToGroup)),
+               Section(type: .more, items: yoursAndMore.more.map(stringToGroup))
```

Comment:
I would remove the bang and turn the `map` in lines 179/180 into a `compactMap`. Same deal in `else`.